MeasureAuthoringTool/mat-drupal
# Theme base atoms

## Description
Provides styles for the base atom components of the site: text, links, buttons, headings, form elements, etc. These are following the "CMS Newsroom" styleguide PDF, and "MAT Website Mockups" PDF.

## To Test
1. Navigate to theme directory `html/themes/custom/emat/`.
2. `npm install` dependencies
3. `npm start` to run pattern lab
4. Visit the localhost URL it generates.
5. In the top, visit links under the "ATOMS" dropdown to see that they are styled as expected
